demo/demo-springmvc/springmvc-server/src/main/resources/microservice.yaml

```diff
@@ -47,10 +47,11 @@ servicecomb:
   # can download config center from https://cse-bucket.obs.myhwclouds.com/LocalCSE/Local-CSE-1.0.0.zip to test dynamic config
   config:
     client:
-  #    serverUri: http://127.0.0.1:30113
+      serverUri: http://127.0.0.1:30115
```

Comment:
This test is an auto-test in docker and no config center installed. So you can not submit this test code.  
After you finished your local test, you can add test cases without config center. For example, add yaml files to `microservice.yaml`

```
mytest:
  yaml: |
      hostName: wyk
```